site.py

```diff
@@ -0,0 +1,86 @@
+import os
+import subprocess
+import urllib
+import json
+import time
+import random
+
+from flask import Flask, render_template, url_for, request, redirect, send_from_directory
+
+
+ALLOWED_EXTENSIONS = set(['csv'])
+app = Flask(__name__)
+app.config['UPLOAD_FOLDER'] = 'tmp/'+ str(random.randint(10000, 99999))+'/'
+
+# Confirm that a provided file has an acceptable extension.
+# In this case, '.csv' is the only allowed extension.
+def allowed_file(filename):
+    return ('.' in filename) and (filename.rsplit('.', 1)[1] in ALLOWED_EXTENSIONS)
```

Comment:
Keep this simple, remove `ALLOWED_EXTENSIONS` and just do `return filename.endswith('.csv')`.